search/search_indexes.py

```diff
@@ -30,6 +31,12 @@ def index_queryset(self, using=None):
         """Records to check when updating entire index."""
         return self.get_model().objects.all()
 
+    def prepare_text(self, obj):  # pylint: disable=no-self-use
```

Comment:
@ShawnMilo ok, btw, is this related/helpful ? http://stackoverflow.com/a/2741778/4992607